app/models/preservation_policy.rb

```diff
@@ -26,4 +31,16 @@ def self.seed_from_config
   def self.default_preservation_policy
     find_by!(preservation_policy_name: Settings.preservation_policies.default_policy_name)
   end
+
+  def self.cached_default_preservation_policy_id
```

Comment:
question:  do the callers need to know it's cached?  would we be okay with naming this method `default_pp_id`, for example, or `default_policy_id` or `default_id`?  

I would also be in favor of changing `default_preservation_policy` method name to `default_policy` ...

Reply:
+1 to changing it to `default_policy`

Reply:
sure, `default_policy_id` and `default_policy` it is.  i sorta like making it explicit that it's cached, but i don't feel strongly about that, and am happy to leave it out of the name if others would prefer.  could also leave a comment.

Reply:
I'm in favor of leaving a comment about caching if, as it looks likely, we call this `default_policy`, if only b/c `id_cache` seems like a really terse method name to me, but I wouldn't demand a comment.

Reply:
i left a comment in my follow-on commits